### PR TITLE
Extend Neteller config to support new Neteller PaySafe Payments API

### PR DESCRIPTION
**Pull request by iamdevice**

### CODE REVIEW

**Review by rensftw — DISMISSED** [review]
Schema looks good to me, but I haven't tested it locally

File: openapi/components/schemas/GatewayAccountConfig/Neteller.yaml [hunk 1]
```diff
@@ -9,25 +9,13 @@ allOf:
         type: object
         description: Neteller credentials object
         properties:
-          clientId:
+          paysafePaymentsApiUsername:
             type: string
-            description: Neteller App Client ID
-          clientSecret:
+            description: Neteller App Paysafe Payments API Username (PRIVATE Key)
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Neteller.yaml:14]:
```suggestion
            description: Neteller App Paysafe Payments API Username
```
?

Comment (iamdevice) [reply openapi/components/schemas/GatewayAccountConfig/Neteller.yaml:14]: This API has public and private keys. We need only private keys.

Comment (ashkarpetin) [reply openapi/components/schemas/GatewayAccountConfig/Neteller.yaml:14]: Sorry, I didn't understand, the field is `paysafePaymentsApiUsername` how it is related to private key?

Comment (iamdevice) [reply openapi/components/schemas/GatewayAccountConfig/Neteller.yaml:14]: Private and public keys are credentials and each of them has username and password.

Comment (iamdevice) [reply openapi/components/schemas/GatewayAccountConfig/Neteller.yaml:14]: https://developer.paysafe.com/en/additional-documentation/neteller-migration-guide/api/#/introduction/api-keys

File: openapi/components/schemas/GatewayAccountConfig/Neteller.yaml [hunk 2]
```diff
@@ -9,25 +9,13 @@ allOf:
...
+          paysafePaymentsApiPassword:
             type: string
-            description: Neteller App Client Secret
-            format: password
-          webhookSecretKey:
-            type: string
-            description: Neteller Webhook Secret Key
+            description: Neteller App Paysafe Payments API Password (PRIVATE Key)
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Neteller.yaml:17]:
```suggestion
            description: Neteller App Paysafe Payments API Password (private key)
```

File: openapi/components/schemas/GatewayAccountConfig/Neteller.yaml [hunk 3]
```diff
@@ -9,25 +9,13 @@ allOf:
         type: object
         description: Neteller credentials object
         properties:
-          clientId:
+          paysafePaymentsApiUsername:
```
Comment (slavcodev) [inline openapi/components/schemas/GatewayAccountConfig/Neteller.yaml:12]: Any reason to add suffix "paysafePayments" to this property?

Comment (iamdevice) [reply openapi/components/schemas/GatewayAccountConfig/Neteller.yaml:12]: Named the same as an original option in Neteller settings from merchant website: `Paysafe Payments API`

**Review by ashkarpetin — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Pull request merged**